src/integrationtests/java/com/aws/greengrass/integrationtests/BridgeTest.java

```diff
@@ -166,4 +169,150 @@ void GIVEN_mqtt5_and_mapping_between_local_and_iotcore_WHEN_local_message_receiv
 
         subscribeCallback.getLeft().get(AWAIT_TIMEOUT_SECONDS, TimeUnit.SECONDS);
     }
+
+    @TestWithMqtt5Broker
+    @WithKernel("mqtt5_local_to_iotcore_retain.yaml")
```

Comment:
nit: can we name it ****_retain_as_published?  just to not cause confusion w/ basic retain